gammapy/scripts/analysis.py

```diff
@@ -0,0 +1,134 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Session class driving the high-level interface API"""
+import copy
+import jsonschema
+import logging
+from ..utils.scripts import read_yaml
+from astropy import units as u
+from pathlib import Path
+
+__all__ = ["Analysis"]
+
+log = logging.getLogger(__name__)
+CONFIG_PATH = Path(__file__).resolve().parent / "config"
+SCHEMA_FILE = CONFIG_PATH / "schema.yaml"
+
+
+class Analysis:
+    """The Analysis class drives an analysis working session using the high-level interface API.
```

Comment:
Always put a short one-line summary, then an empty line, and then extra text in the docstring.
The summary line shows up in API docs summary tables.